### PR TITLE
BI-6753 Message context appended to logging

### PR DESCRIPTION
**Pull request by mattch1**
Added context to loging and log maps as specified on the jira ticket.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java [hunk 1]
```diff
@@ -67,11 +67,12 @@ public void readAndProcess() {
...
                 logMap.put("Message Consumer ID", id);
-                logger.info(String.format("%s - Message deserialised successfully", id), logMap);
+                var messageId = deserializedMsg.getMessageId();
+                logger.infoContext(messageId, String.format("%s - Message deserialised successfully", id), logMap);
 
-                messageProcessorService.processMessage(deserializedMsg);
+                messageProcessorService.processMessage(id, deserializedMsg);
 
-                logger.info(String.format("%s - Message offset %s processed, committing offset", id, msg.getOffset()));
+                logger.infoContext(messageId, String.format("%s - Message offset %s processed, committing offset", id, msg.getOffset()));
```
Comment (ICoxCH) [inline src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java]: I know its not strictly to do with this pr, but it's a logging change so makes sense to do it here.

Could you add to this message, and the Message Offset Retrieved message, the Partition as well as the offset?

Something like "Message Offset: %s, Partition: %s retrieved"
As in staging and live offset alone isn't useful information.

File: src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImplTest.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ void testReadNoMessage() throws ServiceException {
 
         messageConsumer.readAndProcess();
 
-        verify(messageProcessorService, times(0)).processMessage(any());
+        verify(messageProcessorService, times(0)).processMessage(eq(MESSAGE_CONSUMER_ID), any());
```
Comment (ICoxCH) [inline src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImplTest.java]: I would use any(), any() here as we are verifying it is NOT called at all. So better to use vague matchers.

Comment (mattch1) [reply src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImplTest.java]: Changed

File: src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java (hunk 1 above)
Comment (mattch1) [reply src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java]: Updated this to include a partition.

**Review by ICoxCH — APPROVED** [review]

**Pull request merged**